balance.go

```diff
@@ -11,6 +11,43 @@ type TransactionStatus string
 // "application_fee_refund", "transfer", "transfer_cancel", "transfer_failure".
 type TransactionType string
 
+// TransactionSourceType consts represent valid balance transaction sources.
+type TransactionSourceType string
+
+const (
+	// TransactionSourceAdjustment is a constant representing a transaction source of dispute
+	TransactionSourceDispute TransactionSourceType = "dispute"
+
+	// TransactionSourceFee is a constant representing a transaction source of application_fee
+	TransactionSourceFee TransactionSourceType = "application_fee"
+
+	// TransactionSourceCharge is a constant representing a transaction source of charge
+	TransactionSourceCharge TransactionSourceType = "charge"
+
+	// TransactionSourceRefund is a constant representing a transaction source of refund
+	TransactionSourceRefund TransactionSourceType = "refund"
+
+	// TransactionSourceTransfer is a constant representing a transaction source of transfer
+	TransactionSourceTransfer TransactionSourceType = "transfer"
+
+	// TransactionSourceReversal is a constant representing a transaction source of reversal
+	TransactionSourceReversal TransactionSourceType = "reversal"
+)
+
+// TransactionSource describes the source of a balance Transaction.
+// The Type should indicate which object is fleshed out (eg. Fee or Charge)
```

Comment:
Same sort of copy paste problem. Since we have so many potential object types, maybe just remove the part in parenthesis.